enhancements/ingress/set-delete-http-headers.md

```diff
@@ -782,6 +765,26 @@ For this reason, the `host` header is in the deny list of headers that are prohi
 
 Please note that a custom `host` header may be specified via RouteSpec. Please refer to `Open Questions` for more details.
 
+### Cookie and Set-Cookie
+There are three reasons we blocked `Cookie` and `Set-Cookie`:
+
+1. There were no use cases mentioned for these headers in [RFE](https://issues.redhat.com/browse/RFE-464).
+
+2. The cookies that HAProxy sets are used for session tracking, to map a client connection to a particular 
+    backend server.
+
+   The cookie name is an MD5 hash value derived from the backend's name (similarly, the cookie value is a hashed value
+   derived from the backend server's endpoint). 
+   
+   So the cookie names that we would want to forbid are all the possible values
+   of the MD5 hashing algorithm, but it's extremely unlikely that someone would accidentally choose the same cookie name
+   that HAProxy used. 
+  
+   However, if there is an interference caused by setting these then either the HAProxy's session affinity might not work correctly,
+   or HAProxy might overwrite the cookie.
```

Comment:
```suggestion
   We don't want to risk interfering with HAProxy's session affinity, nor restrict HAProxy's ownership of a cookie.
```

Reply:
or HAProxy might overwrite the cookie. sound more better than HAProxy's ownership of a cookie.

Reply:
What do you suggest?  I wanted to clarify what an "interference" is, as this is not a generally used tech term.

Reply:
We don't want to risk interfering with HAProxy's session affinity, nor restrict HAProxy's ownership of a cookie. -- added this